src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: gray;
+}
+
+.list {
```

Comment:
- [ ] As I mentioned above in my first inline comment about sticking with the requirement as much as possible, your design does not match the given design in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). which makes your design `inconsistent` and unprofessional because as a developer it is very important to stick with the requirement, which will help you to become a more experienced, professional and detail-oriented developer. Please have a look at the attached image and table. 

### Requirement
![Screenshot from 2023-08-26 14-52-27](https://github.com/IvonneBenitesRodriguez/todo-list-project/assets/66476329/380022c1-2fec-4c63-a15d-c5bf370bd72a)

## Note 📓 
**Don't misunderstand the requirement of the project, as per the requirement you should implement the minimal feature of the project as per guidance but the design should be the same.**

Reply:
Hello @skyv26 
I modified this section according your suggestions, 
Thanks again,